.travis.yml

```diff
@@ -27,3 +27,12 @@ deploy:
   on:
     repo: schibsted/AccountSDKIOSWeb
     branch: master
+
+notifications:
+  email:
+    if: branch = master
+    recipients:
+      - anna.zamora@schibsted.com
+      - daniel.echegaray@schibsted.com
```

Comment:
Is it possible to use github groups or something?

Reply:
need to check github groups :P